.circleci/config.yml

```diff
@@ -288,7 +288,8 @@ workflows:
           <<: *prod_and_release_jobs
           name: e2e-production-legacy
       - waiter:
-          <<: *angular_jobs
+          requires:
+            - build
```

Comment:
question: What's the difference between this and adding a second `waiter` in the array? e.g.

```yaml
- waiter:
  <<: *angular_jobs
- waiter:
  <<: **prod_and_release_jobs
```

Reply:
the requires: build is just something that was missing before

removing the <<: <job> just means it will run always instead of specifically for both types of jobs (those are our only two jobs)